src/Elastic.Apm/Report/PayloadSender.cs

```diff
@@ -63,23 +59,20 @@ public async void StartWork()
 						new JsonSerializerSettings { ContractResolver = new CamelCasePropertyNamesContractResolver() });

```

Comment:
`Json.NET` caches quite heavily per contract resolver, lets make this a field on the class and reuse it.